modules/managed-identity/user-assigned-identity/main.bicep

```diff
@@ -0,0 +1,57 @@
+@description('Optional. Name of the User Assigned Identity.')
+param name string = guid(resourceGroup().id)
```

Comment:
Does this need to be more unique? What if I use this module twice in the same RG?

Reply:
It's a fair point. Can't say I'm a big fan of auto-generated names anyways. Will remove it